docs/contributing/Pull_Requests.mdx

```diff
@@ -81,3 +81,7 @@ You can [resolve conversations](https://docs.github.com/en/pull-requests/collabo
 Once you've addressed all our feedback by making code changes and/or started a followup discussion, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review) from each maintainer whose feedback you addressed.
 
 Once the feedback is addressed, and the PR is approved, we'll ensure the branch is up to date with `main`, and merge it for you.
+
+### How to Ask
```

Comment:
Suggestion to make it a bit more clear. What do you think about going with one of these?

```suggestion
### Asking for Help
```

```suggestion
### Asking Questions
```

Reply:
That's true. I agree. Asking Questions is a much more clearer title.